attractors/attractors_panel.ipynb

```diff
@@ -26,13 +26,17 @@
     "import numpy as np, pandas as pd\n",
     "from numba import jit\n",
     "\n",
-    "@jit\n",
-    "def trajectory(fn, x0, y0, a, b=None, c=None, d=None, e=None, f=None, n=1000000):\n",
+    "@jit(nopython=True)\n",
+    "def trajectory_coords(fn, x0, y0, a, b, c, d, e, f, n):\n",
     "    x, y = np.zeros(n), np.zeros(n)\n",
     "    x[0], y[0] = x0, y0\n",
     "    for i in np.arange(n-1):\n",
     "        x[i+1], y[i+1] = fn(x[i], y[i], a, b, c, d, e, f)\n",
-    "    return pd.DataFrame(dict(x=x,y=y))"
+    "    return x, y\n",
+    "\n",
+    "def trajectory(fn, x0, y0, a, b=None, c=None, d=None, e=None, f=None, n=1000000):\n",
```

Comment:
Using a,b,c,d,e,f as arguments for all functions was to match Numba's requirement for a fixed argument list. Now that we are using two functions already, it might be better to accept *args here in this function's argument list, and then raise an error if there are too many.  But it's probably not worth worrying about; I'm just recording that sentiment here for posterity.